pipelines/h4c/post_processing/v7_interleave/post_lstbin_frate/h4c_idr1_filtering_allbands_for_lstbin_pspec_nocleanup_frate_filter.toml

```diff
@@ -70,6 +71,13 @@ stride_length="${Options:chunk_size}"
 args=["{basename}", "${Options:include_diffs}", "${Options:label}",
     "${Options:chunk_size}"]
 
+[EXTRACT_AUTOS]
+prereqs = "PRE_CHUNK"
+prereq_chunk_size="all"
+chunk_size="all"
```

Comment:
I'm not sure this is the right way to do it. I think it's probably better to take inspiration from the FRF section, where we set `chunk_size=1` and set the `stride_length` to the number of files in a night. I think invoking it as-is would cause makeflow to generate one `EXTRACT_AUTOS_blah.sh` file for every file. If I'm understanding the intended workflow correctly, I'd think that we only want to run the script once.

Reply:
(See new toml)

Reply:
Hmmm. I'm not sure whether it's possible to export environment variables into the toml. Tbh, I'm very fuzzy on the details of how the chunking stuff even works with makeflow, and just figured that we could mimic what the FRF code does since it operates on an entire night's worth of data. Maybe @jsdillon can provide further insights?